app/assets/stylesheets/action_page.css.scss

```diff
@@ -434,6 +433,10 @@ h3.privacy-notice-header {
   }
 }
 
+.email-signup {
+  text-align: left;
```

Comment:
Can also use the bootstrap class "text-left" https://getbootstrap.com/css/#type-alignment